src/Resources/contao/modules/ModuleRegistration.php

```diff
@@ -312,6 +312,7 @@ protected function compile()
 		// Add the groups
 		foreach ($arrFields as $k=>$v)
 		{
+			// TODO: why is this necessary?
 			$this->Template->$k = $v; // backwards compatibility
```

Comment:
Because of this commit: https://github.com/contao/core-bundle/commit/0fb309d4439b21d96cdadb9535923ad7ff7b1e41